modify SCIM plugin (#106)
pluginsGLPI/doc: source/scim/okta.rst

- you can now **close** this window and **save** your configuration

- Always in **provisionning**, you can edit and select the possible actions for updating your user data.

.. Warning::
   We recommand to unselect **Sync Password** and use `OAuth SSO <https://glpi-plugins.readthedocs.io/en/latest/oauthsso/okta.html>`_ to authenticate your user.

.. figure:: images/scim-23.png
   :alt: Enable update data
   :scale: 67 %

The last step is to assign your application to users

Synchronising all users
~~~~~~~~~~~~~~~~~~~~~~~

- Go to admin console
- Select **Directory > Groups**
- Select **Everyone**
- In **Applications**, click **Assign applications**
- Click on assign on your SCIM application
- Click **Save and Go Back**

.. figure:: images/scim-24.png
   :alt: assign everyone
   :scale: 100 %

Synchronising selected groups and users
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

- Go to admin console
- Select **Directory > People**
- Select the **User** you want to import
- In **Applications**, click **Assign applications**
- Click on assign on your SCIM application
- Click **Save and Go Back**

Repeat this step for all users and groups you want to import.

.. Important::

>    See the procedure for setting up the `OAuth SSO <https://glpi-plugins.readthedocs.io/en/latest/oauthsso/okta.html>`_ plugin to authenticate users on GLPI  to authenticate your user.

Reviewer: ```suggestion
   See the procedure for setting up the `OAuth SSO <https://glpi-plugins.readthedocs.io/en/latest/oauthsso/okta.html>`_ plugin to authenticate users in GLPI.
```

   :scale: 100 %

- you can now **close** this window and **save** your configuration

- Always in **provisionning**, you can edit and select the possible actions for updating your user data.

.. Warning::
   We recommand to unselect **Sync Password** and use `OAuth SSO <https://glpi-plugins.readthedocs.io/en/latest/oauthsso/okta.html>`_ to authenticate your user.

.. figure:: images/scim-23.png
   :alt: Enable update data
   :scale: 67 %

The last step is to assign your application to users

Synchronising all users
~~~~~~~~~~~~~~~~~~~~~~~

- Go to admin console
- Select **Directory > Groups**
- Select **Everyone**
- In **Applications**, click **Assign applications**
- Click on assign on your SCIM application
- Click **Save and Go Back**

.. figure:: images/scim-24.png
   :alt: assign everyone
   :scale: 100 %

Synchronising selected groups and users
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

- Go to admin console
- Select **Directory > People**
- Select the **User** you want to import
- In **Applications**, click **Assign applications**
- Click on assign on your SCIM application
- Click **Save and Go Back**

Repeat this step for all users and groups you want to import.

.. Note::

>    It seems that OKTA does not offer an automatic way of assigning users to an application if you choose to synchronise groups and selected users.

Author: ok, i'll delete this section
